### PR TITLE
Add Integration Tests for Calculator Controller

### PR DESCRIPTION
**Pull request by hrogat**
Automated changes after successful tests.

## Agent Update

- **[Coder]** Created CalculatorControllerIntegrationTest with MockMvc tests for the /sum endpoint, covering valid inputs, edge cases, and negative inputs.
- **[Tester]** All tests passed including the new CalculatorControllerIntegrationTest. The integration tests for the /sum endpoint work correctly with valid inputs, edge cases, and negative inputs.

### CODE REVIEW

File: src/test/java/com/tomwey2/calculator/CalculatorControllerIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    @Test
+    void testSumWithNegativeInputs() throws Exception {
+        mockMvc.perform(get("/sum")
+                        .param("a", "-1")
+                        .param("b", "-1"))
+                .andExpect(status().isOk())
+                .andExpect(content().string("-2"));
+    }
+
```
Comment (hrogat2) [inline src/test/java/com/tomwey2/calculator/CalculatorControllerIntegrationTest.java]: add a test with a positive and negative parameter

**Review by hrogat2 — CHANGES_REQUESTED** [review]

**Comment by hrogat** [comment]
**Automated Update:**

Automated changes after successful tests.

## Agent Update

- **[Coder]** Updated CalculatorControllerIntegrationTest to use parameters 5 and 7 for valid inputs test, and added a new test case with positive and negative parameters as requested in the PR review feedback.
- **[Tester]** All tests passed successfully. The CalculatorControllerIntegrationTest now includes the requested changes: using parameters 5 and 7 for valid inputs test, and adding a new test case with positive and negative parameters. Total tests run: 6, Failures: 0, Errors: 0, Skipped: 0.
